front/src/hooks/useAuthorization.ts

```diff
@@ -1,21 +1,24 @@
 import { useAuth0 } from "@auth0/auth0-react";
 
 interface IAuthorizeProps {
-  requiredAbp?: string[];
+  // requiredAbps: [abp1, abp2] => user must have abp1 AND abp2
+  // requiredAbps: [[abp1, apb2], abp3] => user must have (abp1 OR abp2) AND abp3
+  requiredAbps?: (string | string[])[];
   minBeta?: number;
 }
 
 export function useAuthorization() {
   const { user } = useAuth0();
 
-  const authorize = ({ requiredAbp, minBeta }: IAuthorizeProps) =>
-    user && (
-      user["https://www.boxtribute.com/roles"].includes("boxtribute_god") || (
-      (requiredAbp?.every((abp) => user["https://www.boxtribute.com/actions"].includes(abp)) ??
+  const authorize = ({ requiredAbps, minBeta }: IAuthorizeProps) =>
+    user &&
+    (user["https://www.boxtribute.com/roles"].includes("boxtribute_god") ||
```

Comment:
Small nit: Move this accessor `"https://www.boxtribute.com/roles"` to a constant somewhere?